packages/design-system/src/components/css-value-list-item.stories.tsx

```diff
@@ -0,0 +1,165 @@
+import { css, styled, theme } from "../stitches.config";
+import { CssValueListItem } from "./css-value-list-item";
+import { Label } from "./label";
+import { SmallToggleButton } from "./small-toggle-button";
+import {
+  EyeconOpenIcon,
+  EyeconClosedIcon,
+  SubtractIcon,
+} from "@webstudio-is/icons";
+import * as React from "react";
+import { SmallIconButton } from "./small-icon-button";
+import {
+  FloatingPanelPopover,
+  FloatingPanelPopoverContent,
+  FloatingPanelPopoverTrigger,
+} from "./floating-panel-popover";
+import { StorySection, StoryGrid } from "./storybook";
+
+export default {
```

Comment:
If this is a demo of a component from Figma, please put under "libaray" (add `title` here same as in other files)